server/compactor/src/main/java/org/apache/accumulo/compactor/Compactor.java

```diff
@@ -736,8 +736,10 @@ public void run() {
           final FileCompactorRunnable fcr =
               createCompactionJob(job, totalInputEntries, totalInputBytes, started, stopped, err);
 
-          final Thread compactionThread =
-              Threads.createThread("Compaction job for tablet " + job.getExtent().toString(), fcr);
+          // TODO KEVIN RATHBUN exists within a while(!shutdown) loop so thread is repeatedly
+          // recreated. No need to be critical. If a single job fails, that's okay.
+          final Thread compactionThread = Threads.createNonCriticalThread(
+              "Compaction job for tablet " + job.getExtent().toString(), fcr);
```

Comment:
exists within a while(!shutdown) loop so thread is repeatedly recreated. No need to be critical. If a single job fails, that's okay.